src/pkgcore/ebuild/eapi.py

```diff
@@ -715,3 +715,5 @@ def _combine_dicts(*mappings):
     )),
     ebd_env_options=eapi7._ebd_env_options,
 )
+
+latest_eapi = eapi8
```

Comment:
I'm not a huge fan of 'latest' here; pkgcore's original intent was to support varying EAPI's, including developmental.  Is a developmental one 'latest' ?

TL;DR: can we call this `latest_pms_eapi` ?

Reply:
Hell, we could have `pms_eapis = [obj1, obj2, obj3, ...]` and allow folks to just do `pms_eapis[-1]` to get latest.  Thoughts?

Reply:
I agree. This was a draft PR, more as a "WIP" branch for the future. You can ignore this PR from any review for the near future.